src/reducers/columnStateHelper.js

```diff
@@ -323,8 +323,8 @@ function reorderColumn(oldState, reorderData) {
       columnKey: columnKey,
       columnWidth: width,
       originalLeft: left,
-      columnsBefore: [],
```

Comment:
They're removed, but columnBefore and columnAfter (not columns) are new and needed